tools/clang/tools/dxc/dxc.cpp

```diff
@@ -274,6 +311,25 @@ int DxcContext::ActOnBlob(IDxcBlob *pBlob, IDxcBlob *pDebugBlob, LPCWSTR pDebugB
 
   CComPtr<IDxcBlobEncoding> pDisassembleResult;
 
+  // SPIRV Change Starts
+#ifdef ENABLE_SPIRV_CODEGEN
+  if (m_Opts.GenSPIRV) {
+    CComPtr<IDxcLibrary> pLibrary;
+    IFT(m_dxcSupport.CreateInstance(CLSID_DxcLibrary, &pLibrary));
+
+    if (DisassembleSpirv(pBlob, pLibrary, &pDisassembleResult)) {
+      if (!m_Opts.AssemblyCode.empty()) {
+        WriteBlobToFile(pDisassembleResult, m_Opts.AssemblyCode);
+      } else {
+        WriteBlobToConsole(pDisassembleResult);
```

Comment:
You can add a test to the /utils/hct/hcttestcmds.cmd file to have it succeed on errorlevel 0 or if we find the "not available" string in the output.

Reply:
Done. Not sure what I've done is the typical batch script way. I tried to do the `findstr` in the first if check but it seems `errorlevel` will not be updated at all. Haven't figured out why.